sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/models/ece-integration.model.ts

```diff
@@ -0,0 +1,48 @@
+import { DisbursementValue } from "@sims/sims-db";
+
+export const ECE_SENT_TITLE = "CONFIRMATION REQUEST";
+
+/**
+ * Result of a file uploaded to SFTP on ZONE B network.
+ */
+export interface ECEUploadResult {
+  generatedFile: string;
+  uploadedRecords: number;
+}
+
+/**
+ * Represents a single line in a ECE request file.
```

Comment:
"Represents a single line" is not accurated because of the `disbursementValues`, right?